<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>treePainter</class>
 <widget class="QWidget" name="treePainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="insertContent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{

background-color: rgb(255, 255, 255);
       border-radius:10px;border: 2px groove gray;
}</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="eraseContent">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{

background-color: rgb(255, 255, 255);
       border-radius:10px;border: 2px groove gray;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertNode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(230,230,250);
        color: black; 
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>insert</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/insert.png</normaloff>:/new/prefix1/insert.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="eraseNode">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(230,230,250);
        color: black; 
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>erase</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/erase .png</normaloff>:/new/prefix1/erase .png</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="findContent">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{

background-color: rgb(255, 255, 255);
       border-radius:10px;border: 2px groove gray;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findNode">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(230,230,250);
        color: black; 
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>find</string>
   </property>
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/new/prefix1/find.png</normaloff>:/new/prefix1/find.png</iconset>
   </property>
  </widget>
  <widget class="QTextBrowser" name="oldTreeInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="newTreeInfo">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
